packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -27,13 +27,23 @@ export interface DescriptionListProps extends Omit<React.HTMLProps<HTMLDListElem
   /** Sets the number of columns on the description list */
   columnModifier?: {
     default?: '1Col' | '2Col' | '3Col';
+    sm?: '1Col' | '2Col' | '3Col';
     md?: '1Col' | '2Col' | '3Col';
     lg?: '1Col' | '2Col' | '3Col';
     xl?: '1Col' | '2Col' | '3Col';
     '2xl'?: '1Col' | '2Col' | '3Col';
   };
+  /** Indicates how the menu will align at screen size breakpoints. Default alignment is set via the position property. */
+  layouts?: {
```

Comment:
We use `orientation` in the sidebar component with a value of stack (vertical) and split (horizontal). Do you think this prop should match? I'm wondering if we should update the values to be vertical/horizontal to match, too. WDYT?

Reply:
Yeah that sounds good to me

Reply:
Updated the prop name and value order.